src/test/java/uk/gov/pay/api/it/GetPaymentIT.java

```diff
@@ -114,6 +114,38 @@ private void assertPaymentWithMetadata(ValidatableResponse response) {
                 .body("metadata.surcharge", is(1.23f));
     }
 
+    @Test
+    public void getPaymentWithMotoThroughConnector() {
+        connectorMockClient.respondWithChargeFound(CHARGE_TOKEN_ID, GATEWAY_ACCOUNT_ID,
+                getConnectorCharge()
+                        .withMoto(true)
+                        .build());
+
+        ValidatableResponse response = getPaymentResponse(CHARGE_ID);
+
+        assertCommonPaymentFields(response);
+        assertConnectorOnlyPaymentFields(response);
+        assertPaymentWithMoto(response);
+    }
+
+    @Test
+    public void getPaymentWithMotoThroughLedger() {
+        ledgerMockClient.respondWithTransaction(CHARGE_ID,
+                getLedgerTransaction()
+                        .withMoto(true)
+                        .build());
+
+        ValidatableResponse response = getPaymentResponse(CHARGE_ID, LEDGER_ONLY_STRATEGY);
+
+        assertCommonPaymentFields(response);
+        assertPaymentWithMoto(response);
+    }
+
+    private void assertPaymentWithMoto(ValidatableResponse response) {
+        response
+                .body("moto", is(true));
```

Comment:
can this can all go on one line please? this line break looks weird.

Reply:
Done